pheps/phep-0002.md

```diff
@@ -0,0 +1,126 @@
+<a name="reference-implementation"></a>
+In a sense, this PHEP is itself a reference implementation.
+
+A reference implementation is not always necessary, but is strongly suggested for Standards Track PHEPs. In that case it would include a reference to an existing or new PyHC project which is compliant with the proposed standard and a description of how it complies. The description should point out where in the compliant project's repository the standard is implemented and how it fulfils the requirements.
+
+The reference implementation must be completed before any PHEP is given status "Final", but it need not be completed before the PHEP is accepted. While there is merit to the approach of reaching consensus on the specification and rationale before writing code, the principle of "rough consensus and running code" is still useful when it comes to resolving many discussions of API details.
+
+The final implementation must include test code and documentation appropriate for the relevant PyHC project(s).
+
+# Rejected Ideas
+<a name="rejected-ideas"></a>
+Throughout the discussion of a PHEP, various ideas will be proposed which are not incorporated. Those rejected ideas should be recorded along with the reasoning as to why they were rejected. This helps record the thought process behind the final version of the PHEP and prevents people from bringing up the same rejected idea again in subsequent discussions. The author should also solicit community input on a PHEP's idea before drafting the PHEP; substantial suggestions rejected during this process should also be captured.
+
+This should not be a full recap of the back-and-forth discussion, but only a summary of the significant ideas which were not selected.
+
+# Open Issues
+<a name="open-issues"></a>
+While a PHEP is in draft, ideas can come up which warrant further discussion. Those ideas should be recorded so people know that they are being thought about but do not have a concrete resolution. Suggestions may also arise from preliminary conversations before the PHEP is drafted, which should be captured if they are not resolved before writing begins. This helps make sure all issues required for the PHEP to be ready for consideration are complete.
+
+# Footnotes
+<a name="footnotes"></a>
+A collection of footnotes cited in the PHEP, and a place to list non-inline hyperlink targets. Other notes may be included in this section as appropriate, or it may be excluded.
+
+# Revisions
+This section contains a brief summary of changes by revision number, including the date of approval.
+
+Revision 1 (pending): Initial approval.
+
+# Copyright
+<a name="copyright"></a>
+The text following this paragraph should be included verbatim. The BibTeX code should be updated with the PHEP number (once it is assigned) in both the citation key and the `number` tag, the author, title, and year. Once a DOI is reserved, the BibTeX citation should be updated with the DOI for the specific revision of the PHEP.
```

Comment:
```suggestion
The text following this paragraph should be included verbatim. The BibTeX code should be updated with the PHEP number (once it is assigned) in both the citation key and the `number` tag, the author, title, and year. Once a DOI is reserved for the document (not the repository), the BibTeX citation should be updated with the DOI for the specific revision of the PHEP.
```

Reply:
Was the vote to approve assuming the inclusion of this change?

Reply:
Yeah.

Reply:
Excellent. Just pushed a minor revision with this change and the link to the first vote.